### ISSUE TITLE
training stops in few seconds and no checkpoint file created

### ISSUE THREAD

**Issue by Ahmad-noborders** [xcmyz/FastSpeech#80]
hello @xcmyz, i am training on custom data and followed all these steps

- [x] Download and extract LJSpeech dataset.

- [x] Put LJSpeech dataset in data.
- [x] Unzip alignments.zip.
- [x] Put Nvidia pretrained waveglow model in the waveglow/pretrained_model;
- [x] Run python preprocess.py.
- [x] * if you want to calculate alignment, don't unzip alignments.zip and put Nvidia pretrained Tacotron2 model in the Tacotron2/pretrained_model

preprocess.py creates train.txt file and also creates files in  mels folder
but train.py stops in only 5 to 6 seconds with following outpout

> Model Has Been Defined
> Number of FastSpeech Parameters: 51380417
> Defined Optimizer and Loss Function.
> 
> ---Start New Training---

and after that no  checkpoint file created model_new folder is empty. any help is highly appreciated
thanks

**Comment by xcmyz** [comment]
try "unzip `alignments.zip`"

**Comment by Ahmad-noborders** [comment]
@xcmyz as i mentioned i have already tried with  unzip alignments. but still same thing happens

**Comment by xcmyz** [comment]
I didn't get enough information to locate the problem. You need check your dataloader.
